pkg/attribute/mutableBag.go

```diff
@@ -305,73 +319,79 @@ func GetBagFromProto(attrs *mixerpb.Attributes, globalDict []string) (*MutableBa
 	// TODO: fail if the proto carries multiple attributes by the same name (but different types)
 
 	var buf *bytes.Buffer
+	log := func(format string, args ...interface{}) {}
+
 	if glog.V(2) {
 		buf = pool.GetBuffer()
-		log(buf, "Creating bag from wire attributes:\n")
+		log = func(format string, args ...interface{}) {
+			fmt.Fprintf(buf, format, args...)
```

Comment:
suggestion: all the log() calls end with "\n". perhaps add that detail here, and save having to add it to each call.

maybe something like:
```
if buf.Bytes()[buf.Len()-1] != '\n' {
  buf.WriteByte('\n')
}
```